frontend/src/components/c-summary-charts.vue

```diff
@@ -627,6 +656,15 @@ export default defineComponent({
       } = this;
       // Deep copy to ensure changes in zoom (e.g. toggle state) won't affect summary, and vice versa
       const clonedUser = JSON.parse(JSON.stringify(user));
+
+      if (this.optimiseTimeline) {
+        const optimisedMinimumDate = this.getOptimisedMinimumDate(user);
+        const optimisedMaximumDate = this.getOptimisedMaximumDate(user);
+
+        if (optimisedMinimumDate != null) since = window.getDateStr(optimisedMinimumDate);
+        if (optimisedMaximumDate != null) until = window.getDateStr(optimisedMaximumDate);
+      }
+
```

Comment:
Currently, using the tab zoom on an optimised/trimmed timeline doesn't work. The zoomed-in chart uses the duration of the optimised timeline min and max dates instead of the user selected zoom duration. I think removing the logic I highlighted here may help, as the `since` and `until` values passed to `opneTabZoom()` already use the expected since and until values for the zoom function.